packages/core/src/interpreter.ts

```diff
@@ -554,14 +556,14 @@ export class Interpreter<
 
   private sendTo(
     event: SCXML.Event<AnyEventObject>,
-    to: string | number | ActorRef<any>
+    to: string | ActorRef<any>
   ) {
     const isParent = this.parent && to === SpecialTargets.Parent;
     const target = isParent
       ? this.parent
       : isActorRef(to)
       ? to
-      : this.children.get(to);
+      : this.children.get(to.startsWith('#_') ? to.slice(2) : to);
```

Comment:
Just a general wondering - should this be handled here at the end or maybe normalized when resolving `send()`? It has implications on what will be displayed in the visualizer etc. 

I don't have an opinion about this one - just something I'm raising for additional consideration.

Reply:
You're right, this should probably be resolved in `send()`.

IIRC the reason for `#_` is because the `src` can be a URL instead of an ID, and this is how it's distinguished.

In the future, in XState, a URL `src` can be converted to an actor with its URL as an ID (XState would not care that it's a URL).